feat(content): Korath 'nra'ret (Ravager)

Saugia opened the pull request:
**Content (Artwork / Missions / Jobs)**

![ravager](https://user-images.githubusercontent.com/93169396/185995904-b0b6f71a-178d-4acb-9ff5-3b0ca0e19a08.png)

## Summary
This PR is to add the Korath 'nra'ret (previously known as the Ravager) to the game's base data for use in future content. While it was *possibly* going to be added in https://github.com/endless-sky/endless-sky/pull/7088, the original idea was to add it as a separate PR, similar to the Raider Mark XLII, as a later-on developed Korath Warship. The design is one of Nomadic's from https://github.com/endless-sky/endless-sky/pull/4987, with several adjustments discussed in discord. 

The name has been given to it through https://github.com/endless-sky/endless-sky/pull/7101, where we are updating all Korath names similarly.

![ravager](https://user-images.githubusercontent.com/93169396/185995930-51d92eed-adf4-4694-95cf-57b09f4772b9.png)

The Korath 'nra'ret is more of a mid-long range artillery platform than it is a beefy frontline warship, unlike the upcoming KWS warships and other warships. Dropping Shield and Hull durability for more heavy weapon usage and speed/maneuverability, it's to be the premiere platform of the Firelight Missile as a true missile boat- several gun ports surrounding it to launch broadside barrages of Firelights at enemies. On top of this, it makes great use of Heavers as artillery distancing weapons and has a pair of Fire-Lances for any close range shenanigans.

An Automata variant "Crippler" (same concept as the Raider Mk. 42's version) is a weapon platform of Automata Tech using Detainers and Piercers.

These builds effectively make the 'nra'ret a late-game monster for the Korath to use against the Remnant. @Zitchas has more about this in his future chapters for the Remnant. 

The idea was to get this in before 0.9.15 earlier, but we also planned on putting it in the Worldship PR if we could. Since that PR is looking to need more time, this could be a way to fit in the Ravager as a teaser for what is to come.

## Save File
This save file can be used to play through the new mission content:
N/A

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/72
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/285

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,75 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "The Korath Ravager is powerful artillery ship armed with numerous missile ports, designed to rain fire down on enemies before they are close enough to burn them up with their powerful heat lances."
```
```suggestion
	description "The Korath Ravager is a powerful artillery ship armed with numerous missile ports, designed to rain fire on enemies before they are close enough to burn them up with their powerful heat lances."
```

tibetiroka reviewed (changes requested):
Just a minor fix for the ship description; there isn't much grammar to check here.

Saugia commented:
>Just a minor fix for the ship description; there isn't much grammar to check here.

It's currently a placeholder anyway, no worries. Thank you.

Hurleveur commented:
I love how this ship just comes in guns blazing in all directions!
Also since we're using aiming missiles we don't need the AI to aim right with each gun port, smart.
That said I expected this ship to be better in terms of mobility than it is, because if you don't have engines you should be able to take hits (or in this case I guess its able to deliver some good hits, but isn't it meant to fight from a range?)
We could maybe just use a few banishers and call it a day though.

Saugia commented:
It's not necessarily meant to fight at a range, which is why it also has powerful Infernos. It's a glass-cannon, so its purpose is to deal a heavy amount of damage while the rest of the fleet blockade around it. It's not supposed to take a lot of damage either. While one of these by itself may be weaker, a few together is very strong, and a few with heavier escorts is even stronger.

Hurleveur commented:
Okay so its really not meant to be fighting alone is the thing, got it.
Because I can really see it being bullied by remnant ships if it is alone or by 2 even.

Saugia commented:
Yes, definitely. One or two easily start to get overwhelmed by Remnant system defenses, but 3+ start to cause some casualties, and then under the idea that they come with more of a fleet, even more.

roadrunner56 commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,75 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "The Korath Ravager is a powerful artillery ship armed with numerous missile ports, designed to rain fire on enemies before they are close enough to burn them up with their powerful heat lances."
```
Wait, does the enemy burn up the Ravager with their heat lances, or does the Ravager burn up the enemy with their heat lances?

Hurleveur replied:
I think `they are close` could be changed by `getting close enough`

roadrunner56 reviewed (changes requested):
Disclaimer: I'm not an art guy, but it looks like the flares for the Ravager are too big for the exhaust. The flares go over the borders of the exhaust pipes, which I think needs fixing. Otherwise the description looks good. Can't speak for stats though.

ravenshining commented:
wait did we not include an inferno dredger in stage I?

my memory is foggy, but did we determine that the ravages could fit infernoes  +  fire lance in the space allotted for the detainer + warder variant w/ o adding extra?

for the record i'm not opposed to it having leave a port or two empty in order to fit infernoes or detainers if that balances it as a high-end T2 HWS.  the focus of this one's build should be long range strikes and i feel infernoes detract from that contrast wrt to the 619 (formerly conqueror)

warp-core commented:
> for the record i'm not opposed to it having leave a port or two empty in order to fit infernoes or detainers if that balances it as a high-end T2 HWS. the focus of this one's build should be long range strikes and i feel infernoes detract from that contrast wrt to the Ra'gru Ik 618 (formerly conqueror, formerly KWS-B)

It has 520 weapon capacity; there's not much it doesn't have space for.
It's also definitely not a T2 heavy warship, certainly not with these stats.

ravenshining commented:
I mean, equivalency wise, not in terms of pigeonholing hull strenght and speed to arbitrarily restrictive definitions

Saugia commented:
>wait did we not include an inferno dredger in stage I? 

I believe we agreed upon that the Worldships+ would be the ideal platforms for it, but no, there are no Stage I ships that have infernos. 

>my memory is foggy, but did we determine that the ravages could fit infernoes + fire lance in the space allotted for the detainer + warder variant w/ o adding extra? 

Yep; default variant fits 2 infernos,2 fire-lances, and 6 firelight banks. The automata fits 2 detainers, 2 warders, 2 piercer launchers and 4 firelight banks.

warp-core commented:
> I mean, equivalency wise, not in terms of pigeonholing hull strength and speed to arbitrarily restrictive definitions

Equivalency of what?
It's got lots of weapon space, but that's about it. The Emerald Sword only has marginally less weapon space, and significantly more hitpoints but is not regarded as tier 2 by anyone.
The base mass is inbetween the 512 and 349. The outfit space is a decent chunk lower, but still close, and certainly still in the heavy warship category. The weapon capacity is very high (though, neither of the two variants actually use more than 400 of it). Again, heavy warship territory (arguably super-heavy). But, the Emerald Sword has bigger values in all of these categories. The most significant indicator of tech level really is total hitpoints. Of course, there is the possibility that a ship might be designed to deliberately have less durability than is possible for its size in exchange for having better performance in other areas, like more outfit space or weapon capacity, and this certainly has more weapon capacity, but I'm not sure the ~80 extra over the 349, alongside the >100 reduction in outfit space really corresponds to such an enormous reduction in hitpoints at the same tech level.

> i'm deliberately only including one significant digit when I say T2

Rounding your tiers so broadly seems to me to eliminate any notion of usefulness from the concept.

ravenshining commented:
> Equivalency of what? ... weapon space ... hitpoints ... base mass ... outfit space ... weapon capacity 

I think you're missing the forest for the trees again.  Equivalent in *overall performance* to another T2 HWS, or a T3 LWS.

>  The most significant indicator of tech level really is total hitpoints. 

I emphatically and vehemently disagree with that philosophy.  Again.

Tiers are meant to compare the overall capabilities of ships, not dictate specific stats.

> > i'm deliberately only including one significant digit when I say T2
> 
> Rounding your tiers so broadly seems to me to eliminate any notion of usefulness from the concept.

It gives you a large margin for error.  If the Ravager were only as strong as a shield beetle, it'd be too weak.  If it matches the big Kahet thing, a 512, a 349, or a Punisher, it's good.  We don't really have examples of T2.75+ HWS to say what's too strong unfortunately.

ravenshining commented:
> I believe we agreed upon that the Worldships+ would be the ideal platforms for it, but no, there are no Stage I ships that have infernos. No Dredgers use blaze-pikes either, as we left that to Raiders so that Dredgers are not just warships also, and have more unique loadaouts/roles.

Ideal =/= only

Hm, I missed the deletion of fire dredgers, as I was trusting you to handle that and only skimming myself.   I would have rather had them be rare encounters in home space and the Wastes, but no matter, we can bring them back in another PR.

It's variants like the Ground Assault Dredger that *severely* abuse outfit expansions that I take exception to; one or two expansions to fit better weapons on a high-end variant is fine for the Korath.

warp-core commented:
> I think you're missing the forest for the trees again. Equivalent in overall performance to another T2 HWS, or a T3 LWS.

Well, it's definitely not. One gets destroyed by a Hurricane without too much trouble. Two do manage to win, though, it does take a while and only one survives (they would've won much sooner and probably both survived without Derpy's heat nerf, though).

Testing against a Punisher, it's the same story, one loses, two just about win (only one survives).

ravenshining commented:
> Well, it's definitely not. One gets destroyed by a Hurricane without too much trouble. Two do manage to win, though, it does take a while and only one survives (they would've won much sooner and probably both survived without Derpy's heat nerf, though).
> 
> Testing against a Punisher, it's the same story, one loses, two just about win (only one survives).

Ah, okay, the testing we did on this was prior to the heat nerf.  Sounds to me like this ship needs buffed, then.

And, if it might ease your mind a bit, having this ship have a higher hull strength than usual for Exile ships would fit in-lore

warp-core commented:
It'll need an enormous buff for it to be able to defeat a Hurricane, 349, or Punisher.

ravenshining commented:
So let's get buffin'!

I haven't reviewed the stats for this yet at all, only the hardpoints, which is part of why this was going to be left until Stage III or later (although admittedly I to prefer to leave stat work to number-minded people like you and Ejo where possible).

ravenshining commented:
Speaking of, @EjoThims can you take a look and help get this up to spec?  I know you're good at that sort of thing.

Saugia commented:
>Ideal =/= only

I should have been more clear- ideal meaning more fitting, since Dredgers are more multi-role vessels with cargo/crew focus, *not* warships. There was a long discussion about this and that is where we ended up with it, so I'm not in favor of just half-hazardly changing our minds, but that's for a different discussion in a different place.

>Hm, I missed the deletion of fire dredgers, as I was trusting you to handle that and only skimming myself. I would have rather had them be rare encounters in home space and the Wastes, but no matter, we can bring them back in another PR.

I would have to be against this as stated above. We decided Dredgers are, as above, very utility focused and took them away from the warship role. If we give them infernal weapons, that would straight up put them by Raiders once again, which was the whole point of not doing so. The various diggers, grab-strikes, banishers, and even shunt strikes make much more sense on them. Leaving blaze pikes to raiders and blaze pikes and infernos to larger ships is much more fitting because they are the more front-heavy aggressive vessels. Once again, a discussion for a different place, as I really hate going on tagents on PR's not about those tangents. 

>And, if it might ease your mind a bit, having this ship have a higher hull strength than usual for Exile ships would fit in-lore

This is completely against the concept of the Ravager we settled on. It devalues the speed and intense weapon load of it and therefore reverts it into a stat stick warship. The intention was not to create another "big powerful korath warship that has everything- speed, tankiness, and weapon room". The tankiness and weapon bit is left to the KWS's, respectfully, while the Ravager is a glass-cannon that can't take as many hits as a Dredger could. 

>left until Stage III or later....

There's nothing this really hurts at all. We have a lot of capable eyes, and implementing this ship sooner helps Zitchas with his content writing and gives an extra little taste (if he does get a mission going) if we aren't able to push the Worldship PR. Since the plan was to introduce it separately like the Raider 42, I don't think it's a big deal.

ravenshining commented:
> > And, if it might ease your mind a bit, having this ship have a higher hull strength than usual for Exile ships would fit in-lore
> 
> This is completely against the concept of the Ravager we settled on. It devalues the speed and intense weapon load of it and therefore reverts it into a stat stick warship. The intention was not to create another "big powerful korath warship that has everything- speed, tankiness, and weapon room". The tankiness and weapon bit is left to the KWS's, respectfully, while the Ravager is a glass-cannon that can't take as many hits as a Dredger could.
> 
> I'd be okay if it got a little love for defense, but not to make it as on par with its speed and weapon load.

Oh, I don't mean to make it more resillient than the Conqueror at all.  I was just looking for a way to compromise with warp-core about its place in the hierarchy, if it had a higher hull:shield ratio

Saugia commented:
Ah. I see, I thought it was because it seemed too low (which was sort of the point). Warps ideas are either the mass is too heavy and needs to be lighter, or the defenses need to be buffed a bit accordingly.

ravenshining commented:
It can be buffed in other, more role-relevant ways

Ideally it should out-range opponents and trash one something equivalent in size... but then it runs out of ammo.  Unfortunately the AI currently doesn't support such behaviour, but IIRC we had close enough to that point before the heat nerf

beccabunny reviewed (changes requested):
I honestly never liked the Ravager's appearance; it's extremely flat, doesn't really look of the same species as any other Korath ship and even then it's less detailed than something like the Raider. However, if the other people really like it I won't be the one blocking the PR

ravenshining commented:
> I honestly never liked the Ravager's appearance; it's extremely flat, doesn't really look of the same species as any other Korath ship and even then it's less detailed than something like the Raider. However, if the other people really like it I won't be the one blocking the PR

The idea is that it's more in line with a big chaser, than a big raider.  The single-hull construction is not something the Korath can really achieve with warships... until the story bestows them with some recovered manufacturing capacity.

It is pretty flat though, I'd support some thickening.

warp-core commented:
> Oh, I don't mean to make it more resillient than the Conqueror at all. I was just looking for a way to compromise with warp-core about its place in the hierarchy, if it had a higher hull:shield ratio

I don't have a huge issue with where it is, I just wanted to make sure it was clear that it cannot keep up with the T2 heavy warships.

EjoThims reviewed (changes requested):
Wow, BIG changes from the NV version. Especially that drastic shift to fixed guns.

I really like using the vented look around the edges for firelight placements though! :)

I do really like how this solidifies it's role compared to other Exile ships, especially in fleet compositions.

I honestly can't remember if the HP numbers are different (or how much so) from NVs, but either way I think this goes way, way overboard in reducing its survival when it is supposed to be a sign of the Exiles moving back up towards T2 proper.

I do agree that tiers shouldn't ever be -just- stat stick checks, but the numbers as they are now (especially combined with the gun/turret mix) make this much more comparable to a Hai MW than a brand spanking new Exile HW.

Overall, even once addressing the above, I think this needs to lean much more heavily on -how and why- it's different from other Exile ships in filling its role as a combat focused, HW sized escort/pursuit vessel.

Doing so helps drive home the point that this ship does not represent the advancement of current Exile capabilities/priorities (the way the newer Raider might), but rather the explicit ability to pursue new/different priorities as well.

Changes to bunks, crew, and cargo should be more drastic to help illustrate that difference, but in no way should the specific numbers I've suggested be considered set in stone.

Total lack of fighters also does help with that, and I definitely think that should be a hard line with it we do not change. :)

ravenshining commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
+		"required crew" 277
+		"bunks" 319
+		"mass" 1260
+		"drag" 12.3
```
Drag could def be lower, crew and bunks idk, mass is already p. low

Saugia replied:
I'll take it down ton 11.3 for now.

ravenshining commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
+		"cargo space" 110
+		"outfit space" 915
+		"weapon capacity" 520
+		"engine capacity" 177
```
Reversers seem a bit inappropriate for the role and sprite, to me.

EjoThims replied:
That's fair, but then I think we shouldn't stop the turning more than a single step even as we increase -either- the thruster or the AB size.

I think I'm personally leaning towards planetary thrust and AB with lunar steering.

I like the idea of having afterburners as big as the primary thrust to give this thing a REAL kick when it needs it. :)

Saugia replied:
Fortunately, dropping from Planetary Steering to Lunars, and raising from Lunar AB to Planetary, is just *one* tone more needed in outfit space- which is perfect as the default variant had 1 ton remaning. Will look at the Automata variant shortly.

ravenshining commented on data/korath/korath variants.txt:
```diff
@@ -413,6 +413,41 @@ ship "Korath Raider Mark XLII" "Korath Raider Mark XLII (Crippler)"
+
+		"Afterburner (Lunar Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Planetary Class)"
```
Ahh, I wasn't aware there was an AI to better use reversers in the works.  In that case it's probably fine, but like other Korath ships, I think the sprite should have explicit nozzles if that's the case.

EjoThims replied:
This one

https://github.com/endless-sky/endless-sky/pull/6483

ravenshining commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "The Korath Ravager is a powerful artillery ship armed with numerous missile ports, designed to rain fire on enemies before getting close enough to burn them up with their powerful heat lances."
```
'nra'ret-class

warp-core commented:
> mass is already p. low

I'm not sure what you're comparing to.
Mass is about normal, maybe a tiny bit high, for ships with this outfit space.
It's lower than average for ships with this weapon capacity.
It's high for the heat dissipation.
It is ridiculously high for ships with this these hit points (even after Ejo's suggested hit point buff). Maybe a human ship with this durability would have this much mass, but Hai and Remnant ships get this sort of health for much less mass. Ejo's suggestion brings it to about 2% more health than a Shield Beetle, which is only 650 base mass. Granted, it's has 117 less outfit space and 187 less weapon space, but I don't think that is enough for the 600 tonne increase in mass to be reasonable).
I'm actually inclined to reduce the mass further than Ejo has suggested (even after their suggested health increase, which is in line with something I suggested on Discord), though significant increases to outfit space would reduce this inclination.

I think an extra hardpoint for a Warder and an appropriate increase in outfit space (doesn't need a boost to weapon space, it has plenty spare already) is reasonable. A boost to engine space (but not so much outfit space) to give it more flexibility also seems reasonable to me.

This is dependent on the role, characteristics, and behaviour you, Saugia, and Zitchas have in mind. If you want it to stay slow, that's fine, leave the mass, ~but I think it needs more health to not suck in that case~.

I think 0.7 heat dissipation is already a little high, I don't think Ejo's suggested mass reduction is enough for 0.8 to be reasonable. I'll get back on specifics there in the next few hours.

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
+	gun 98 75 "Firelight Missile Bank"
+		angle 170
+	turret -26.5 -11 "Korath Inferno"
+	turret 26.5 -11 "Korath Inferno"
```
Hey I mean buffing the Ravager I'm not against at all :P

Saugia commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,76 @@ ship "Korath Raider Mark XLII"
+		"bunks" 319
+		"mass" 1260
+		"drag" 12.3
+		"heat dissipation" .7
```
This one I'm still iffy about, since it's a very high heat dissipation for a Korath ship. It seems to run just fine at .7 but I'll give everyone some time to throw around suggestions

Saugia commented:
After committing those suggestions, the ones up in the air are still:

Heat dissip from 0.7 to 0.8
Drag (currently down to 11.3 from 12.3)

The extra turret slot I will add once I'm done doing some more model edits.

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,75 @@ ship "Korath Raider Mark XLII"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 3760000
```
Should probably cost more than the 42.
```suggestion
		"cost" 24760000
```

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,75 @@ ship "Korath Raider Mark XLII"
+		"hull" 9850
+		"required crew" 193
+		"bunks" 221
+		"mass" 1120
```
This brings the acceleration to a little lower than the Shield Beetle instead of a little lower than a Model 512.
The turn rate is now only slightly lower than a Hurricane instead of a decent chunk less.
```suggestion
		"mass" 979
```

warp-core reviewed (commented):
I don't think the heat dissipation should be increased from 0.7.

Saugia commented:
Okay, latest changes:

Changed Lunar Afterburner to Planetary, and Planetary *Steering* to Lunar on default variant. Bumped the 108 Firelight missiles to 120 (108 wasn't the full capacity of the 6 launchers). Added a third turret mount for a Warder on the default. 

Added a single ton to the default's weapon space so the Automata Variant can now fit 2 Repeater Turrets with 1 Warder with the new extra turret, as opposed to the 2 Warders from before. Changed Lunar Afterburner to Planetary, Planetary *Thruster* to Lunar, and kept the Planetary Steering (which is now one above the default variant). 

Once we get the WS PR in with the Firestorms, I'll make a Firestorm Variant. If this PR is smaller and faster to get in first, I will make a Firestorm Variant for the WS PR.

W1zrad commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. It's size and non-modular nature also means it must be constructed ...
```
```suggestion
	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
```

W1zrad commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. It's size and non-modular nature also means it must be constructed ...
+	description "The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances to continue burning enemies in close quarters as we ...
```
```suggestion
	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances to continue burning enemies in close quarters as well."
```

Saugia commented:
I've decided to go ahead and add the Ravager as a very rare spawn alongside the Korath's automata development in the late game so that it is present with this PR. This should provide some fun speculation from the community as to what it is and its purpose while creating suspense for Stage II (should that not make it into 9.15 in time), or to support Stage II as another addition (if it does make it in).

Saugia commented:
github pls

roadrunner56 commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances to continue burning enemies in close quarters as w ...
```
```suggestion
	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances so that it is able to burn enemies in close quarters as well."
```

roadrunner56 commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
```
```suggestion
	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
```

Hurleveur reviewed (approved):
This really isn't my area of expertise, but the ship feels nice when piloting it.
The sprite is very Korathy, ~~though for some reason I can see a face in it but that's always the case, isn't it~~
Very nice to have a ship that's different as in it doesn't only have forward facing guns.
Also the exiles were missing a ship with many guns.

roadrunner56 reviewed (commented):
Very minor comment, and more of a nitpick than anything substantial.

roadrunner56 reviewed (approved):
Description looks good.

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
```
The first sentence is a bit too long for my taste; should be fine otherwise.

Saugia replied:
Eh, it shouldn't really be a problem then. Raven is known for her extravagant writing ;P

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances so that it is able to burn enemies in close quarte ...
```
Maybe heat lances should be capitalized to match the outfit name, but I guess technically this also works.

Saugia replied:
Since heat lances are descriptors here and not referencing a specific outfit/title/name, I think this is fine.

Saugia replied:
Now that I think about it, it's not the same as saying "missile" when referring to things, since "heat lances" are more specific. I'll use the caps in this case.

Saugia replied:
Actually I'll just say Fire-Lances.

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances so that it is able to burn enemies in close quarte ...
```
Optional suggestion for readability.
```suggestion
	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances making it able to burn enemies in close quarters as well."
```

Saugia replied:
How about just "and two smaller heat lances allowing it to burn enemies in close quarters."?

Saugia replied:
@roadrunner56 Thoughts?

roadrunner56 replied:
Sounds good.

tibetiroka reviewed (commented):
Some minor issues, mainly about phrasing. Kind of optional.

warp-core commented on data/wanderer/wanderers middle.txt:
```diff
@@ -3328,6 +3328,10 @@ event "wanderers: exiles have drones"
 		add variant 4
 			"Model 32"
 			"Model 16" 5
+		add variant
+			"Korath Ravager"
+		add variant
+			"Korath Ravager (Crippler)"
```
Putting it here means these ships won't get added to the fleet for older saves.
Allowing older saves that have already had this event happen would require you to create an invisible mission that requires this event to have happened and triggers an event that adds just these variants to the fleet.

Saugia replied:
Ah, I see. Do you think it's fine to do that so this is possible, with the invisible mission?

I will leave the Automata production part to you on your Moonbeam PR, since you're addressing that with the KIV's there too.

Saugia replied:
I guess for the sake of this PR, I *won't* introduce it then, and leave that to Zitchas.

warp-core reviewed (commented):
So, the heat dissipation is still kind of high.
Mereti ships, however, actually have higher heat dissipation values at similar masses.
So, if, narratively, this ships incorporates some of whatever gives the Mereti ships that, I don't see any problems.

I'm not sure this is entirely compatible with adding this ship at the same time as the automata, though. You'd expect some lead time after acquiring the ability to manufacture those and being able to produce a totally new ship using some features that you only acquired alongside those other ships, even in very small numbers. ~But, putting it on a delay from that at this point in a way that older saves still get it might be more effort than the slight narrative improvement of having these start production a little later is worth, especially when we want to change that rollout anyway and making it more complex here would just cause more problems there~.

Alternatively, I think a heat dissipation of 0.63 is reasonable without necessarily having Mereti tech, at least in comparison to the Raider 42.

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -339,7 +339,7 @@ ship "Korath Ravager"
 	"final explode" "final explosion large"
 	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
-	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances so that it is able to burn enemies in close quarte ...
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances allowing to burn enemies in close quarters."
```
```suggestion
	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller heat lances allowing it to burn enemies in close quarters."
```
It could also be something like `allowing burning enemies` if you prefer that.

Saugia replied:
That sounds a bit odd to me. I think we're fine with the "allowing it to burn".

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+			"hull damage" 2200
+			"hit force" 6600
+	outfits
+		"Korath Inferno" 2
+		"Korath Warder"
+		"Korath Fire-Lance" 2
+		"Firelight Missile Bank" 6
+		"Firelight Missile" 120
+
+		"Systems Core (Large)"
+		"Fuel Processor" 2
+		"Plasma Core"
+		"Triple Plasma Core"
+		"Liquid Sodium Cooler"
+		"Korath Repeater Rifle" 219
+
+		"Afterburner (Planetary Class)"
+		"Steering (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Jump Drive"
```
```suggestion
		"Langrage Hyper-Heaver" 2
		"Korath Warder"
		"Korath Fire-Lance" 2
		"Firelight Missile Bank" 6
		"Firelight Missile" 190
		"Firelight Missile Rack" 7

		"Systems Core (Large)"
		"Systems Core (Medium)"
		"Fuel Processor" 3
		"Triple Plasma Core"
		"Liquid Sodium Cooler"
		"Korath Repeater Rifle" 219

		"Afterburner (Planetary Class)"
		"Steering (Lunar Class)"
		"Thruster (Planetary Class)"
		"Jump Drive"
```

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+		angle 190
+	gun 98 75 "Firelight Missile Bank"
+		angle 170
+	turret -32.5 -53.5 "Korath Inferno"
+	turret 32.5 -53.5 "Korath Inferno"
```
```suggestion
	turret -32.5 -53.5 "Langrage Hyper-Heaver"
	turret 32.5 -53.5 "Langrage Hyper-Heaver"
```

warp-core commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+		"fuel capacity" 900
+		"cargo space" 50
+		"outfit space" 943
+		"weapon capacity" 444
```
```suggestion
		"weapon capacity" 407
```
Still leaves space for the Crippler variant (though, the default variant is probably superior anyway).

warp-core commented:
I suppose the description ought to change as well.

ravenshining commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,77 @@ ship "Korath Raider Mark XLII"
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
+	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller Fire-Lances allowing it to burn enemies in close quarters."
+
```
There are *other* new designs, just not warship designs.  Updated weapons per discord & warp's suggestion above.  Added origin story as other ships in #7101 .  Eliminated some wordiness.

```suggestion
	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly unique warship design the Exiles have fielded since the Exodus, but also the first built solely for combat instead of needing to double as a habitat or supply ship. Its size and non-modular nature also means it must be constructed wholesale at the largest Korath facilities, something that, until recently, was generally reserved for the likes of the massive Ik'atilajej and the Korsmanath A'awoj."
	description "	The 'nra'ret-Class Chaser, named in honor of a brilliant weapons engineer who took their own life to avoid having their secrets captured by a hostile alien force, provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units. To this end, it is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of langrage heavers for sustained operations, and two Fire-Lances for emergency use in close quarters."

```

ravenshining commented on data/korath/korath variants.txt:
```diff
@@ -413,6 +413,43 @@ ship "Korath Raider Mark XLII" "Korath Raider Mark XLII (Crippler)"
+	turret "Korath Repeater Turret"
+	turret "Korath Warder"
+
+
```
Very optional, if you want to keep the inferno variant, have something like this pop up only post-automata along with the crippler (as opposed to the heaver being a remnant/exile unlock)

```suggestion
 ship "Korath Ravager" "Korath Ravager (Inferno)"
 		"required crew" 189
		"bunks" 199
		"weapon capacity" 444
	outfits
		"Korath Inferno" 2
		"Korath Warder"
		"Korath Fire-Lance" 2
		"Firelight Missile Bank" 6
		"Firelight Missile" 120

		"Systems Core (Large)"
		"Fuel Processor" 2
		"Plasma Core"
		"Triple Plasma Core"
		"Liquid Sodium Cooler"
		"Korath Repeater Rifle" 219

		"Afterburner (Planetary Class)"
		"Steering (Lunar Class)"
		"Thruster (Planetary Class)"
		"Jump Drive"
	
	gun "Korath Fire-Lance"
	gun "Korath Fire-Lance"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	gun "Firelight Missile Bank"
	turret "Korath Inferno"
	turret "Korath Inferno"
 	turret "Korath Warder"
 

```

Saugia replied:
I'm not too worried about it for now. We might balance it in the future with additional things like hull and shields, but I think I'm okay with saving this for later (likely with the Firestorm Version depending on when Stage II is merged). Thank you though.

ravenshining reviewed (changes requested):
This description, along with Warp-Core's suggested weapon space nerf and refitting for Heavers, would be to my satisfaction.  Inferno variant I leave up to you, not sure if my syntax is even right there since data isn't my thing.

tibetiroka commented on data/korath/korath ships.txt:
```diff
@@ -329,17 +330,17 @@ ship "Korath Ravager"
-	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly new ship design the Exiles have fielded since the Exodus but also the first purposed directly towards combat usage instead of needing to also double as population and supply housing. Its size and non-modular nature also means it must be constructed  ...
-	description "	The 'nra'ret-Class Chaser provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units, and, to this end, is armed with numerous missile ports designed to rain fire on enemies from afar. It also carries a pair of the powerful Korath Infernos and two smaller Fire-Lances allowing to burn enemies in close quarters."
+	description "While perhaps, at first, seeming like a super-sized 'olofez-Class Carried Chaser, the 'nra'ret-Class Chaser is not only the first truly unique warship design the Exiles have fielded since the Exodus, but also the first built solely for combat instead of needing to double as a habitat or supply ship. Its size and non-modular nature also means it must be constructed wholesale at the l ...
+	description "	The 'nra'ret-Class Chaser, named in honor of a brilliant weapons engineer who took their own life to avoid having their secrets captured by a hostile alien force, provides the Exiles with a fast and powerful patrol striker both in support of large raiding fleets and as independent units. To this end, it is armed with numerous missile ports designed to rain fire on enemies from afar ...
```
I assume the `langrage heavers` doesn't use the exact outfit name on purpose.

ravenshining replied:
It seemed unnecessary to me to be that specific, "langrage heavers" flows more naturally into something one would actually say vs. the technical title Langrage Hyper-Heavers.

Zitchas commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,78 @@ ship "Korath Raider Mark XLII"
 	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
 
 
+ship "Korath Ravager"
```
Question: The name here is "Korath Ravager" and the description calls it a "'nra'ret-Class Chaser." I think I must have missed the discussion regarding its name.

Shouldn't this be consistent one way or the other (preferably its actual Korath name, rather than having the Korath prefix)

Zitchas reviewed (approved):
Looks good to me, with just a question about the names. Namely, the ship name and the name in the description don't match. I suspect I just missed some discussion on the name, but I'd appreciate the clarification if that's the case. If this *is* something that we've just all overlooked, I do think we should standardize all the references. If it is the 'nra'ret, then let's just call it that everywhere, rather than calling it one thing in one place, and another thing elsewhere.

Seeing as this is a new ship and not renaming old assets that need to be preserved to avoid breaking stuff, I'd prefer to see all the assets simply calling it by its proper name right from the start. (and not waiting for the Korath Names PR before switching them over, if that is a consideration)

Beyond the name issue, I'm very much looking forward to making use of this in the Remnant storyline. It seems a tad on the un-tough side to me; but that is in line with its intended role as an artillery platform.

I would just like to extend my thanks and appreciation to the Swizzle 6 team that has spent a *lot* of time discussing this and tweaking things, much of it before it ever appeared as a PR. Even if it doesn't look like there's been that much discussion here, this is just the culmination of a lot of people's efforts.

As far as discoverability goes, I should have a mission PR up in the next few days that makes use of this ship in a subtle sort of way. Curious explorers will be able to find it and warn the Remnant about it, but there won't be any significant numbers of them, nor will there be anyone standing around telling the player to go find it, either.

W1zrad commented:
I understand the sentiment for wanting to go ahead and update the name in accordance with #7101, but that PR has yet to be merged. Effectively, it's not in the game yet. It's still proposed, not final. **It feels preemptive to merge this as-is, because then we're acting on the assumption that #7101 is merged without any further deliberation.** #7101 feels like it should remain a conversation separate from this PR, and by using the naming scheme suggested in #7101, now this PR is tied to it.

Thus, I think we should either (1) revert the name to Ravager and update all the names at once, or (2) wait for the name update PR to be merged then merge this with the new name. In the case of option 1, if there's content created (by Zitchas or others) between the time of merging and #7101 being merged, I'm obviously happy to help update the names myself.

EjoThims commented:
Considering derpy just signed off on 7101 with suggestion to also update the not yet used sprite there, I think best to get that updated and merged and then marge this as is.

Saugia commented:
What they said.

ravenshining commented on copyright:
```diff
@@ -1560,6 +1560,13 @@ Copyright: Saugia <https://github.com/Saugia>
 Comment: Derived from works by Michael Zahniser (under the same license) and Lia Gerty (under the same licence).
 
+Files:
+ images/ship/nraret*
+ images/thumbnail/nraret*
```
```suggestion
 images/ship/-nra-ret*
 images/thumbnail/-nra-ret*
```

ravenshining commented on data/korath/korath ships.txt:
```diff
@@ -270,6 +270,78 @@ ship "Korath Raider Mark XLII"
 
 
+ship "'nra'ret"
+	sprite "ship/nraret"
+	thumbnail "thumbnail/nraret"
```
```suggestion
	sprite "ship/-nra-ret"
	thumbnail "thumbnail/-nra-ret"
```

ravenshining reviewed (commented):
In #7088 I've been using hyphens in place of apostrophes (since this character is not meant to represent a contraction, but a glottal stop, as in uh-oh).  it might be good to stay consistent.

Zireael07 commented:
I'm guessing it has a Korath name now to match the other recent PR?

Saugia commented:
Yep
